JeMPI_Apps/JeMPI_API/src/main/java/org/jembi/jempi/api/HttpServer.java

```diff
@@ -924,7 +960,11 @@ private Route createJeMPIRoutes(
                   path(segment(GlobalConstants.SEGMENT_GOLDEN_RECORD_ROUTE).slash(segment(Pattern.compile("^[A-z0-9]+$"))),
                        (goldenId) -> AppConfig.AKKA_HTTP_SESSION_ENABLED
                              ? routeSessionFindExpandedGoldenRecord(actorSystem, backEnd, goldenId)
-                             : routeFindExpandedGoldenRecord(actorSystem, backEnd, goldenId)))));
+                             : routeFindExpandedGoldenRecord(actorSystem, backEnd, goldenId)),
+                    path(segment(GlobalConstants.SEGMENT_FHIR).slash(segment(Pattern.compile("^[A-z0-9]+$"))),
```

Comment:
Let's rather add a new route set in createRoutes() where the prefix is "fhir" instead of "JeMPI" line 973